pom.xml

```diff
@@ -2,7 +2,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>software.amazon.msk</groupId>
 	<artifactId>msk-config-providers</artifactId>
-	<version>0.4.0-SNAPSHOT</version>
+	<version>0.3.1</version>
```

Comment:
please do no change the version. I'll merge it into trunk and into 0.3.1 tag version to maintain proper versioning

Reply:
Done!